Install.bat

```diff
@@ -0,0 +1,199 @@
+@echo off
+setlocal enabledelayedexpansion
+set VENV_DIR=.venv
+set "requirements_file=requirements.txt"
+:main
+set "wrong_python=0"
+cls
+if exist "%VENV_DIR%" (
+    call:cecho red "Previous virtual environment detected. May cause issues if not deleted."
+    echo [33m
+    set /p "choice=Would you like to remove this virtual environment? [Y/N]:"
+    echo [0m
+    set "choice=!choice:~0,1!"
+    set "choice=!choice:~0,1!"
+    if /i "!choice!"=="Y" (
+         rmdir /s /q %VENV_DIR%
+    ) else if /i "!choice!"=="N" (
+        call:cecho red "You have chosen not to remove the current virtual environment. Please note this may cause issues with the installation."
+    ) else (
+        call:cecho red "Invalid choice. Please enter Y or N."
+        goto :again
+    )
+)
+cls
+python --version >nul 2>&1
```

Comment:
What if `python` doesn't work but `python3.10` for example does work? Then this script thinks Python isn't installed

Reply:
oh, so like a PATH issue? I see what you mean. I'll get that resolved. Thanks